lib/googleauth/signet.rb

```diff
@@ -38,6 +38,11 @@ module OAuth2
     # This reopens Client to add #apply and #apply! methods which update a
     # hash with the fetched authentication token.
     class Client
+      def initialize
+        super
+        @refresh_listeners = nil
```

Comment:
This one is not going to work. We're re-opening an existing class `Signet::OAuth2::Client` from the `signet` gem. This `initialize` method will _replace_ the method from the original class, which we don't want to do. (And the `super` calls the initializer on the superclass which happens to be `Object`.) Instead, I think we're going to need to find the uses of `@refresh_listeners` and guard them with, for example, `if defined? @refresh_listeners`.

Reply:
Addressed in https://github.com/googleapis/google-auth-library-ruby/pull/230/commits/f908519d5523db021e5bd17094b5b14ec9b0e2bc.